include/irr/asset/IMeshPacker.h

```diff
@@ -21,13 +18,96 @@ template <typename MeshBufferType>
 class IMeshPacker
 {
 public:
-    virtual std::optional<std::pair<MeshBufferType* ,DrawElementsIndirectCommand_t>> packMeshes(const core::vector<MeshBufferType*>& meshBuffers) = 0;
+    struct AllocationParams
+    {
+        size_t indexBuffSupportedSizeInBytes                    = 2147483648ull; /*  2GB*/
+        size_t vertexBuffSupportedSizeInBytes                   = 4294967296ull; /*  4GB*/
```

Comment:
SSBO is kinda limited to 2GB, so 2GB makes a good default for the data heavy data streams (indices and vertices)